deliver/lib/deliver/options.rb

```diff
@@ -245,6 +245,10 @@ def self.available_options
                                      description: "Metadata: The localised app name",
                                      optional: true,
                                      is_string: false),
+        FastlaneCore::ConfigItem.new(key: :subtitle,
+                                     description: "Metadata: The localised app subtitle",
+                                     optional: true,
+                                     is_string: false),
```

Comment:
This will need a verify block like https://github.com/fastlane/fastlane/pull/9370/files#diff-cec46a69540d57cbb9c8631d43a7a89bR262